Frameworks/Foundation/NSSet.mm

```diff
@@ -554,12 +573,93 @@ - (void)setValue:(id)value forKey:(NSString*)key {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 - (NSString*)descriptionWithLocale:(id)locale {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return [self descriptionWithLocale:locale indent:0];
+}
+
+/**
+ @Status Interoperable
+*/
+- (NSString*)description {
+    return [self descriptionWithLocale:nil indent:0];
+}
+
+- (NSString*)descriptionWithLocale:(id)locale indent:(NSUInteger)level {
```

Comment:
Unfortunately this can't be implemented by wrapping NSArray's descriptionWithLocale:indent: in braces because levels will cause spaces to appear between the first brace "{" and the array's parenthesis "("